core-domain/src/main/java/gohigher/user/oauth2/GoogleOAuth2User.java

```diff
@@ -0,0 +1,35 @@
+package gohigher.user.oauth2;
+
+import java.util.Map;
+
+public class GoogleOAuth2User extends OAuth2UserInfo {
+
+	public GoogleOAuth2User(Map<String, Object> attributes) {
+		super("sub", attributes);
+	}
+
+	@Override
+	public String getOAuth2Id() {
+		return (String)attributes.get(oauth2IdAttributeName);
```

Comment:
```suggestion
		return (String) attributes.get(oauth2IdAttributeName);
```
자동 정렬하면 이렇게 공백이 들어가지 않나요..? 해당 클래스 파일 정렬(Command + option + L)한번 하면 좋을 것 같습니다!!

Reply:
자동 정렬하니 공백이 사라졌습니다!! 저도 공백이 있는 편을 선호하는 편인데 공백이 있는 걸로 맞출까요?
(혹시 해당 부분 자동 정렬 설정하는 법 아실까요...?)